docs/guide/web-simulation.md

```diff
@@ -54,13 +54,18 @@ Then, start a session and simulation servers:
 cd $WEBOTS_HOME/resources/web/server
 ./server.sh start default
 ```
+
+If you have installed Webots with the `.deb` or the `.snap` package, you should:
+ - Copy the [server]({{ url.github_tree }}/resources/web/server) folder in a place where you have read/write rights, so that the server can be launched without administrator privileges.
+ - Open the `default.json file` located in `server/config/simulation/` with you ide/text editor and add the following line between the `{}`: "webotsHome": "/usr/local/webots". This line is used to tell the server where Webots is located
```

Comment:
Here there are some typos:
```suggestion
 - Open the `default.json` file located in `server/config/simulation/` with your IDE or text editor and add the following line between the `{}`: ``"webotsHome": "/usr/local/webots"``.
This line is used to tell the server where Webots is located.

Additionally it would be better to add hyperlinks to all the files and folders.
```